include/RAJA/exec-cilk/forall_cilk.hxx

```diff
@@ -404,7 +404,7 @@ RAJA_INLINE void forall_Icount(cilk_for_exec,
                                Index_type icount,
                                LOOP_BODY loop_body)
 {
-  const Index_type* __restrict__ idx = iseg.getIndex();
+  const Index_type* RAJA_RESTRICTre idx = iseg.getIndex();
```

Comment:
The way we have things set up now, this would only be caught by building with an Intel compiler. We don't do that in travis, correct?